fluidefi/fluidefi-caspernet-aggregator-tools
# Feature/native price service

**Summary of change:**

- Created sync record fetcher class.
- Created all pairs finder class.
- Created native price calculator class.
- Created native price saver class.
- Created native price consumer class.



**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/native-price-service`
```bash
git checkout feature/native-price-service
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm tests have passed.
- Run example.
